spec/lib/calm/api_spec.rb

```diff
@@ -3,6 +3,8 @@
 RSpec.describe Calm::Api do
   # include the helper module
   include Savon::SpecHelper
+  
+  ENV['CALM_ENDPOINT'] = 'http://www.calmhosting01.com/CalmAPI-HullUni/archive/catalogue.asmx?WSDL'
```

Comment:
Wouldn't this be available from the .env file?

Reply:
not necessarily on the spec server?